20-logging.py

```diff
@@ -31,10 +34,11 @@
     t_level = "DEBUG"
     level = logging.DEBUG
 # Set up WARNING and above as stderr, below that to stdout.
-handlers = [
-    logging.StreamHandler(level=logging.WARNING),
-    logging.StreamHandler(level=level, stream=sys.stdout),
-]
+warnhandler = logging.StreamHandler(stream=sys.stderr)
+warnhandler.setLevel(logging.WARNING)
+lowhandler = logging.StreamHandler(stream=sys.stdout)
+lowhandler.setLevel(level)
+handlers = [warnhandler, lowhandler]
```

Comment:
We don't need this in the main code block. Can we only define these handlers in an else block below? Setting up two handlers that we are never going to use seems like overhead.